QUALITY_DECLARATION.md

```diff
@@ -1,4 +1,4 @@
-This document is a declaration of software quality for the `rtest` package, based on the guidelines in [REP-2004](https://www.ros.org/reps/rep-2004.html).
+This document is a declaration of software quality for the `rtest` package, based on the guidelines in [REP-2004](https://reps.openrobotics.org/rep-2004/).
```

Comment:
Oh, cool. Is https://reps.openrobotics.org a modern replacement for https://www.ros.org/reps ?